db/data/regions/AT/pt-BR.yml

```
@@ -5,16 +5,15 @@ pt-BR:
       addresses:
         address1:
           label:
-            default: Rua e número da casa
-            optional: Rua e número da casa (opcional)
+            default: Endereço e número da casa
+            optional: Endereço e número da casa (opcional)
```

I think it should be only "número" in this line and the one belo
```suggestion
            default: Endereço e número
            optional: Endereço e número (opcional)
```

---

Updated!